src/components/builder/values/items-expression.tsx

```diff
@@ -32,7 +32,12 @@ export const ItemsExpression: React.FC = () => {
       }
     >
       <div>
-        <JSONEdit name={NAME} data={value} rows={3} id={htmlId} />
+        <JSONEditor
+          height="15vh"
```

Comment:
`15vh` might not scale well, I'm wondering if there's a way I can set a height corresponding to 3 rows of a `textarea`

Reply:
no, but you can do a workaround like in `src/registry/content/richText.scss`

I'd also really prefer the style changes to be done in SASS :)

Reply:
is there a way to pass the `style` prop? You could use the css var approach: `style={{'--json-edit-rows': 3}}` and have the CSS use `var(--json-edit-rows, 15)`.